bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -31,9 +31,9 @@ metadata:
     categories: Logging & Tracing
     certified: "false"
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2020-12-16T13:37:00+00:00"
```

Comment:
Since operator-sdk 1.26.0, `make bundle` changes the `createdAt` field from the bundle even if it is patched:
https://github.com/operator-framework/operator-sdk/pull/6136

I added a workaround this issue in the Jaeger Operator:
https://github.com/jaegertracing/jaeger-operator/blob/main/Makefile#L356
https://github.com/jaegertracing/jaeger-operator/blob/main/hack/ignore-createdAt-bundle.sh

Reply:
Thank you for the heads-up. I've just submitted the fix.